django/contrib/admin/templates/admin/app_list.html

```diff
@@ -7,6 +7,13 @@
         <caption>
           <a href="{{ app.app_url }}" class="section" title="{% blocktranslate with name=app.name %}Models in the {{ name }} application{% endblocktranslate %}">{{ app.name }}</a>
         </caption>
+        <thead class="visually-hidden">
+          <tr>
+            <th>{% translate 'Model name' %}</th>
```

Comment:
For completeness, add `scope="col"` to each `<th>` so the browser doesn't have to guess the direction of the table heading.

Reply:
Thank you :star: 
I also updated the header "Change link" to "Change list link" as sometimes it says "View" rather than "Change" depending on permissions but (if it's there) it always points to the change list page of that model